modules/assim.sequential/R/Analysis_sda.R

```diff
@@ -139,7 +141,7 @@ GEF<-function(settings, Forecast, Observed, H, extraArg, nitr=50000, nburnin=100
   path.to.models <- file.path(settings$outdir,"SDA","GEF")
   aqq <- extraArg$aqq
   bqq <- extraArg$bqq
-  wts <- extraArg$wts
+  wts <- extraArg$wts/sum(extraArg$wts)
   interval <- NULL
```

Comment:
suggestion :
`if(all(is.na(wts))) wts <- rep(1, length(extraArg$wts)) # if there is no weight then they all have the same weight`

Reply:
The weights need to sum to 1 so that's why I put this here. But, maybe I should move it to the main sda function?

Reply:
I think it's fine here, but I was suggesting to replace NA values in `wts` with some weights which you added your fix.